src/target/riscv/riscv-013.c

```diff
@@ -153,7 +155,9 @@ typedef struct {
 	/* Number of abstract command data registers. */
 	unsigned datacount;
 	/* Number of words in the Program Buffer. */
-	unsigned progbufsize;
+	unsigned int progbufsize;
```

Comment:
```suggestion
	unsigned progbufsize;
```
Please, stick to just using `unsigned`

Reply:
`unsigned int` is preferable, according to style guide.
https://github.com/riscv-collab/riscv-openocd/blob/22bbdd2a5e9388376c8097ac9ea0cbfede3680d6/doc/manual/style.txt#L177

Moreover, Checkpatch will issue a warning.

Reply:
Agree with @en-sc

Reply:
That's right, OpenOCD code style asks for `unsigned int`.

Reply:
oh, in that case, we should change it elsewhere